### PR TITLE
Convert cmds {`.`, `afv[WR]j`}

### PR DESCRIPTION
**Pull request by b1llow**
**Your checklist for this pull request**
- [ ] I've read the [guidelines for contributing](https://cutter.re/docs/contributing/code/getting-started.html) to this repository
- [ ] I made sure to follow the project's [coding style](https://cutter.re/docs/contributing/code/development-guidelines.html)
- [ ] I've updated the [documentation](https://cutter.re/docs/user-docs.html) with the relevant information (if needed)

**Detailed description**

**Test plan (required)**

**Closing issues**

Partially addresses
https://github.com/rizinorg/cutter/issues/2666

### CODE REVIEW

File: src/core/Cutter.cpp [hunk 1]
```diff
@@ -479,24 +457,11 @@ bool CutterCore::asyncCmd(const char *str, QSharedPointer<RizinTask> &task)
...
-
-        if (offset != core->offset) {
-            updateSeek();
-        }
-    });
-
-    return true;
+    auto task = std::unique_ptr<RizinTask>(new RizinFunctionTask(std::move(fcn), true));
+    task->startTask();
+    task->joinTask();
```
Comment (karliss) [inline src/core/Cutter.cpp:464]: I am not too familiar with the task infrastructure @thestr4ng3r  made, but this seems redundant. If `functionTask` is immediately starting the task and waiting for it to finish, you might as well do core lock directly and do the thing you want to do.

Comment (b1llow) [reply src/core/Cutter.cpp:464]: https://github.com/rizinorg/cutter/blob/617c79f97653c52f7d4d36fcb4e5f0a86c94723c/src/core/Cutter.cpp#L556-L562

I imitated what I wrote here, and as you say, maybe there is no need for a Task that is not async at all

Comment (XVilka) [reply src/core/Cutter.cpp:464]: Yes, I think @karliss is right; in this case no need for the task, but I am not very familiar with this feature as well; better to wait for what @thestr4ng3r has to say about this.

Comment (thestr4ng3r) [reply src/core/Cutter.cpp:464]: > If functionTask is immediately starting the task and waiting for it to finish, you might as well do core lock directly and do the thing you want to do.

Not exactly. The core can only be held locked by one thread at a time. But it is possible to do this start+join pattern concurrently multiple times and the running tasks will be scheduled in an interlocked fashion. Also it is possible to break such a running task.

**Review by XVilka — APPROVED** [review]

File: src/core/Cutter.cpp [hunk 2]
```diff
@@ -4033,22 +3968,34 @@ QList<SearchDescription> CutterCore::getAllSearch(QString searchFor, QString spa
...
-                RVA addr = address.toRVA();
-                xref.from = addr;
-                xref.to = addr;
-                if (findWrites) {
-                    xref.from_str = RzAddressString(addr);
-                } else {
-                    xref.to_str = RzAddressString(addr);
-                }
-                xrefList << xref;
+    RzList *vars = rz_analysis_var_all_list(Core()->core()->analysis, fcn);
```
Comment (karliss) [inline src/core/Cutter.cpp]: The values `RZAnalysisVar` are owned by function and not self contained so to ensure that some other task doesn't modify them core lock should be held while processing the list returned by `rz_analysis_var_all_list`.
 
Similar reasoning applies function structure obtained with `functionIn` so  meaning that lock should be started at the start of function.

**Review by karliss — CHANGES_REQUESTED** [review]

**Comment by b1llow** [comment]
```
/home/runner/work/cutter/cutter/build/dist/rz-ghidra-prefix/src/rz-ghidra/src/RizinScope.cpp:10:10: fatal error: rz_build_version.h: No such file or directory
 #include <rz_build_version.h>
          ^~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

**Comment by karliss** [comment]
> ```
> /home/runner/work/cutter/cutter/build/dist/rz-ghidra-prefix/src/rz-ghidra/src/RizinScope.cpp:10:10: fatal error: rz_build_version.h: No such file or directory
>  #include <rz_build_version.h>
>           ^~~~~~~~~~~~~~~~~~~~
> compilation terminated.
> ```

That's probably caused by https://github.com/rizinorg/rz-ghidra/pull/294

**Comment by XVilka** [comment]
@imbillow @karliss, just update Rizin to the latest commit, it should fix the problem.

**Comment by XVilka** [comment]
@imbillow also Cutter seems need an update too:
```
/home/runner/work/cutter/cutter/src/dialogs/AboutDialog.cpp:1:10: fatal error: rz_version.h: No such file or directory
    1 | #include "rz_version.h"
      |          ^~~~~~~~~~~~~~
```

**Review by XVilka — APPROVED** [review]

**Review by karliss — APPROVED** [review]

**Pull request merged**